### PR TITLE
Allow customization of max WS connections.

### LINKED ISSUE #8117: CLI for WS connections capacity.
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: v1.8.2-beta-1b6588c-20171025
> - **Which operating system?**:  Linux
> - **How installed?**: apt-get
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I am running some parity nodes and interacting with them using web3 and WS provider. After ~100 connections I get the error:

 ```
Unable to build WebSocket connection WS Error <Capacity>: Unable to add another connection to the event loop.
```

I just want to check if this is an expected behaviour and if you have any information about concurrent connections capacity because I couldn't find it.

BTW, I am running more than one node for different kind of applications and I get the same issues with most of them.

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #8117

### CODE REVIEW

File: parity/cli/mod.rs [hunk 1]
```diff
@@ -495,6 +495,11 @@ usage! {
 			"--ws-hosts=[HOSTS]",
 			"List of allowed Host header values. This option will validate the Host header sent by the browser, it is additional security against some attack vectors. Special options: \"all\", \"none\".",
 
+			ARG arg_ws_max_connections: (usize) = 100usize, or |c: &Config| c.websockets.as_ref()?.max_connections,
+			"--ws-max-connections=[CONN]",
+			"Maximal number of allowed concurrent WS connections.",
+
+
 		["API and console options – IPC"]
```
Comment (niklasad1) [inline parity/cli/mod.rs]: Remove extra empty line?

**Comment by niklasad1** [comment]
Also I think you need to update the tests for the CI to pass!

**Comment by robdoesstuff** [comment]
Wait a second - did this only allow for settings max WS connections but didn't address the default HTTP connection limit of 250? We don't use WS, only HTTP and have a major need to allow for way more than 250 connections. Major services like mysql often can run right up to the handle limit of 16k or whatever the default is on the OS.

**Comment by Tbaut** [comment]
>Wait a second - did this only allow for settings max WS connections

I guess it's pretty clear from the title. Please create a new issue/PR.

**Comment by 5chdn** [comment]
#8572

**Pull request merged**